tracing/middlewares.go

```diff
@@ -9,14 +9,20 @@ import (
 // InjectHTTPServerSpan returns a go-kit endpoint.Middleware that injects a server
 // span into the `next` context.
 //
-// Note, this depends on the edge context headers already being set on the
-// context object.  This can be done by adding httpbp.PopulateRequestContext as
-// a ServerBefore option when setting up the request handler for an endpoint.
+// Starts the server span before calling the `next` endpoint and stops the span
+// after the endpoint finishes.
+//
+// Note, this function depends on the edge context headers already being set on
+// the context object.  This can be done by adding httpbp.PopulateRequestContext
+// as a ServerBefore option when setting up the request handler for an endpoint.
 func InjectHTTPServerSpan(name string) endpoint.Middleware {
```

Comment:
:no_bell: This function can be just a wrapper for the other one so you don't have to duplicate the logic.

Reply:
Ya, previously it was just a basic wrapper for the `StartSpan` methods so I figured it was fine to just use the two separate functions but now that there's other logic, it's probably better to just have `InjectHTTPServerSpan` call `InjectHTTPServerSpanWithTracer` with a `nil` tracer.